frontend/i18next.scanner/en/translation.json

```diff
@@ -348,5 +348,22 @@
   "Maximum size: {{size}} MB": "Maximum size: {{size}} MB",
   "Registration date:": "Registration date:",
   "Cover image of {{publicName}}": "Cover image of {{publicName}}",
-  "Change cover": "Change cover"
+  "Change cover": "Change cover", 
+  "Comment_sub": "Comment", 
+  "New space": "New space", 
+  "Space updated": "Space updated",
+  "Space deleted": "Space deleted",
+  "Space restored": "Space restored",
+  "Status updated": "Status updated", 
+  "Content updated": "Content updated", 
+  "Item deleted": "Item deleted", 
```

Comment:
Item and content are the same thing? If yes I would say to use "content" to both.

Reply:
I agree that it could be nice to change every instance of "Item" to "Content" if it fits

Reply:
Yes they are actually, I'll update